src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json

```diff
@@ -14,13 +14,29 @@
                 "name": "OsConfigPolicy",
                 "version": "1.0.0",
                 "contentType": "Custom",
-                "contentUri": "<SAS token URL for the artifacts package>",
-                "contentHash": "<file hash for the artifacts package>",
+                "contentUri": "https://gcpoc88.blob.core.windows.net/blobforosconfig/LinuxSshServerSecurityBaselinePolicy.zip?sp=r&st=2024-02-20T19:49:37Z&se=2034-02-21T03:49:37Z&sv=2022-11-02&sr=b&sig=fiZZf8ML276R7PLkpGvYgwHZKIB9e67WU%2BcFAhs%2BACE%3D",
+                "contentHash": "F6874B140C9721248CECE2E803AAC0B94150543158C9EC0DFD688FA5F679BCAF",
                 "configurationParameter": {
-                    "AllowUsers": "Ensure allowed users for SSH access are configured;DesiredObjectValue",
-                    "DenyUsers": "Ensure denied users for SSH are configured;DesiredObjectValue",
-                    "AllowGroups": "Ensure allowed groups for SSH are configured;DesiredObjectValue",
-                    "DenyGroups": "Ensure denied groups for SSH are configured;DesiredObjectValue"
+                    "AccessPermissionsForSshdConfig": "[OsConfigResource]Ensure that permissions on /etc/ssh/sshd_config are configured;DesiredObjectValue",
+                    "Protocol": "[OsConfigResource]Ensure that the SSH protocol is configured;DesiredObjectValue",
+                    "IgnoreHosts": "[OsConfigResource]Ensure that the SSH IgnoreRhosts is configured;DesiredObjectValue",
```

Comment:
Should this be `IgnoreHosts` or `IgnoreRhosts`? Property name and string have conflicting names

Reply:
With R. That's the original check name.